evap/staff/templates/staff_semester_view_course.html

```diff
@@ -60,11 +60,16 @@
                     <span>({{ course.num_reviewed_textanswers|percentage:course.num_textanswers|default:'&mdash;' }})</span>
                 </a>
             {% else %}
-                <p data-toggle="tooltip" data-placement="left" title="{% trans "Text answers" %}">
+                <span data-toggle="tooltip" data-placement="left" title="{% trans "Text answers" %}">
                     <span class="glyphicon glyphicon-comment"></span>
                     <span>{{ course.num_textanswers }}</span>
-                </p>
+                </span>
             {% endif %}
+            <br>
+            <span data-toggle="tooltip" data-placement="left" title="{% trans "Grade documents (Midterm, Final)" %}">
+                <span class="glyphicon glyphicon-file"></span>
+                <span>{% blocktrans with midterm=course.midterm_grade_documents.count final=course.final_grade_documents.count %}M: {{ midterm }}, F: {{ final }}{% endblocktrans %}</span>
+            </span>
```

Comment:
could you move these changes into a separate commit or at least mention it in the commit message?